src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/AuthorizationHelper.cs

```diff
@@ -282,6 +282,30 @@ public static bool VerifyOrgInClaimPrincipal(string org, ClaimsPrincipal user)
             return false;
         }
 
+        /// <summary>
+        /// Verifies a scope claim based on claimsprincipal.
+        /// </summary>
+        /// <param name="requieredScope">Requiered scope.</param>
+        /// <param name="user">Claim principal from http context.</param>
+        /// <returns></returns>
+        public bool ContainsRequieredScope(string requieredScope, ClaimsPrincipal user)
+        {
+            string contextScope = user.Identities?
+               .FirstOrDefault(i => i.AuthenticationType != null && i.AuthenticationType.Equals("AuthenticationTypes.Federation"))
+               ?.Claims
+               .Where(c => c.Type.Equals("urn:altinn:scope"))?
```

Comment:
Let the question mark be on next like together with the dot.

Reply:
done